db/migrate/20180426135126_add_new_fields_to_attendance_events.rb

```diff
@@ -0,0 +1,13 @@
+class AddNewFieldsToAttendanceEvents < ActiveRecord::Migration[5.1]
+  def change
+    add_column :absences, :dismissed, :boolean
```

Comment:
I'm still thinking through whether this is the right shape or not — more information/data to come. 😄